src/server/system_services/bucket_server.js

```diff
@@ -1973,6 +1973,14 @@ function validate_replication(req) {
             throw new RpcError('INVALID_REPLICATION_POLICY', `Rule ${rule_id} destination bucket is invalid`);
         }
 
+        if (rule.log_based_replication_info) {
```

Comment:
`log_based_replication_info` is not part of `rule` anymore. should be moved to out of the for loop